nutils/evaluable.py

```diff
@@ -3206,8 +3278,7 @@ def _multiply(self, other):
     return Ravel(Multiply([self.func, Unravel(other, *self.func.shape[-2:])]))
 
   def _add(self, other):
-    if isinstance(other, Ravel) and equalshape(other.func.shape[-2:], self.func.shape[-2:]):
-      return Ravel(Add([self.func, other.func]))
+    return Ravel(self.func + Unravel(other, *self.func.shape[-2:]))
```

Comment:
This seems to be unrelated to patch 'add _inflations'.

Reply:
fixed